Makefile

```diff
@@ -18,29 +18,33 @@ run_pgstac = docker-compose run --rm \
 image:
 	docker-compose build
 
-.PHONY: docker-run
-docker-run: image
-	$(run_docker)
+.PHONY: docker-run-all
+docker-run-all:
+	docker-compose up
+
+.PHONY: docker-run-sqlalchemy
```

Comment:
explicitly renamed all the targets like `docker-run` to `docker-run-sqlalchemy`, since that's what it does.